core/cloudflow-operator/src/main/scala/cloudflow/operator/event/StatusChangeEvent.scala

```diff
@@ -113,6 +113,8 @@ object StatusChangeEvent extends Event {
                 }
             }
             currentStatuses.get(appId).map(_.toAction(app)).toList
+          // app is in error state, no need to handle updates.
+          case (Some(_), _) => List()
```

Comment:
Instead of leaving the reasoning it in a comment, we can improve by having it in code 🙂 .
You can move this condition above and flip the check:
```scala
case (Some(app), _) if app.status.flatMap(_.appStatus) == Some(CloudflowApplication.Status.Error) ⇒ List()
```

Reply:
Sure